### ISSUE TITLE
Reproducing results

### ISSUE THREAD

**Issue by zehzhang** [SwinTransformer/Video-Swin-Transformer#18]
Hi,

Thanks for the great work.

I'm having the same issue as #5  even when I tested the models with [the same val split](https://github.com/SwinTransformer/storage/releases/tag/v1.0.6).

I played with SwinT and SwinB, and both of them gave 04%~0.5% lower top-1 accuracy than reported. They are still pretty neat but I just want to make sure I am not doing anything wrong.

Would you confirm that the models and the split files uploaded are the correct ones?

Also, if anyone has successfully reproduce it, please kindly comment here about whether there is anything else I need to do besides downloading the models and configs and run the test scripts.

Thanks,

**Comment by hust-nj** [comment]
Hi, before training and testing, the Kinetics400 training and validation datasets we use are preprocessed by making height of video 256, this may cause little difference. You can contact us by v-jianing@microsoft.com for discussing more details.

**Comment by CastleLiang** [comment]
Facing the same problem. I use the annotation files in this repo and test the provided pretrain models without any extra modification. Swin-T and Swin-B achieved 78.4% and 80.1% top-1 accuracy on K400, which seemed slightly worse than the reported results. @zehzhang Have you resolved this issue? If so, could you please share your solution?

**Comment by zehzhang** [comment]
> Facing the same problem. I use the annotation files in this repo and test the provided pretrain models without any extra modification. Swin-T and Swin-B achieved 78.4% and 80.1% top-1 accuracy on K400, which seemed slightly worse than the reported results. @zehzhang Have you resolved this issue? If so, could you please share your solution?

Thanks for confirming the problem. I got similar decrease with SwinT (-0.4% top1 acc) and SwinB pretrained on ImageNet21k (-0.5% top1 acc). I'm reaching out to the other first co-author (referred to by @hust-nj ) and hopefully will figure out what is going on soon. I will keep this thread updated.

**Comment by CastleLiang** [comment]
Hi @zehzhang, thank you much for your prompt reply. Sorry that I forgot to mention that the models I tried were pretrained on ImageNet-1K, but the observations were very similar to yours, i.e., SwinT (-0.4% top-1 acc) and Swin-B (-0.5% top-1 acc). I also checked the preprocessing of the videos and they were consistent with the paper settings (directly using the provided config file). I will keep track of this issue.

**Comment by hust-nj** [comment]
After a careful comparison, we find out that the performance gap is due to the slight difference on data. Our kinetics-400 data with 256 resolution was obtained (with broken video removed) from nonlocal networks which was also used in many other series of works. 

More details and data download link can be found here https://github.com/youngwanLEE/VoV3D/blob/main/DATA.md#kinetics-400, https://github.com/facebookresearch/video-nonlocal-net/issues/67

**Comment by dragen1860** [comment]
@zehzhang Hi, Thanks for your issues. Do you train video-swin from scratch without imagenet-21k, did it drop severely? thank you.
